views/realtime_rest_api.md

```diff
@@ -28,4 +28,235 @@ With regard to the concept of "Conversation" , we have categorized it into three
+
+```sh
+curl -X DELETE \
+  -H "X-LC-Id: {{appid}}" \
+  -H "X-LC-Key: {{masterkey}},master" \
+  https://{{host}}/1.2/rtm/broadcasts/{message_id}
+```
+
+Parameters | Optionality | Description
+---|---|---
+message_id | required | target id, String
+
+return:
+
+blank JSON object `{}`.
+
+### Query the Broadcast Messages
+
+Invoke this API to query all the valid broadcast messages, this interface requires master key to query.
+
+```sh
+curl -X GET \
+  -H "X-LC-Id: {{appid}}" \
+  -H "X-LC-Key: {{masterkey}},master" \
+  https://{{host}}/1.2/rtm/broadcasts?conv_id={conv_id}
+```
+
+Paramters | Optionality | Description
+---|---|---
+conv_id | required | official account ID
+limit | optional | quantity of the messages returned
+skip | optional | number of the messages to skip, for paging.
+
+### Query all the Messages History in the App
+
+This interface requires the master key.
+
+```sh
+curl -X GET \
+  -H "X-LC-Id: {{appid}}" \
+  -H "X-LC-Key: {{masterkey}},master" \
+  https://{{host}}/1.2/rtm/messages
+```
+
+Parameters and the returned value can refer to [One-on-One Chatting/Group Chats query the history messages](realtime_rest_api.html#query history messages) interface.
+
+## Interface requests Frequency Limitation
+
+Operations related to realtime messages invoking REST API has request frequency and quantity limitations (**realtime message SDK API is not influenced**),  details as follows:
+
+* [One-on-One Chatting/Group Chats-send messages](#One-on-One Chatting/Group Chats-send messages)
+* [One-on-One Chatting/Group Chats-modify messages](#One-on-One Chatting/Group Chats-modify messages)
+* [One-on-One Chatting/Group Chats-recall messages](#One-on-One Chatting/Group Chats-recall messages)
+* [Chats Rooms-send messages](#Chats Rooms-send messages)
+* [Chats Rooms-modify messages](#Chats Rooms-modify messages)
+* [Chats Rooms-recall messages](#Chats Rooms-recalll messages)
+* [Official accounts-send message to any user individually](#send message to any user individually)
+* [Official accounts-modify message to any user individually](#modify message to any user individually)
+* [Official accounts-recall message to any user individually](#recall message to any user individually)
+
+#### Limitation
+|Business (per App) |Developer (per App) |
+|---------|---------------|
+|maximum 9000 requests/min, 1800 requests/min by default|120 requests/min|
+
+The Daily usage is calculated based on all the interfaces. LeanCloud will respond with error code 429 if exceeds the limitation until one minutes later. The interface will reboot after that.
+
+The Buisness call ceiling can get mutated in [Dashboard > Messaging > LeanMessage > Settings > Thresholds > Frequency limit for calling API for basic messages][dashboard-rtm-limit].
+According to the daily call frequency peak rate , we do differential pricing. As in the following table:
```

Comment:
According to the daily call frequency peak rate , we do differential pricing. As in the following table: -> You will be billed for daily request frequency rate peak, as below: